### PR TITLE
Introduction tutorial

### PR DESCRIPTION
**Pull request by yvettecook**
#### Adds:
Introduction tutorial with image + gif assets.

Proof reading & feedback very much welcome. 

#### To Do 
* Add 'stretch tasks' to the end of the tutorial
* Add links to other learning resources links (In progress)

### CODE REVIEW

File: 1introduction/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+### Pre-requisites
+
+In order to do iOS development, you will need a Mac.
+
+Download and install Xcode from the App Store. This may take a while — its a big program.
+
+### Creating a Project
+
+1. Open Xcode and click **Create a new Xcode project**.
```
Comment (Abizern) [inline 1introduction/tutorial.md:17]: If you get an popup asking you to install more components, this is created by Xcode itself.

File: 1introduction/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+4. However, it would be even more awesome if our button actually did something.
+
+  To do this we're going to need more than just our storyboard - we'll need a **View Controller**.
+
+  > View controllers are the foundation of your app’s internal structure. Every app has at least one view controller, and most apps have several. Each view controller manages a portion of your app’s user interface as well as the interactions between that interface and the underlying data.
+
+  Think of the view as the puppet, and the view controller as the puppet master. Without something pulling the strings, the view is just a lifeless doll. And without the puppet, the puppet master is just someone doing silly voices to empty air.
+
+  ![muppets gif](https://giphy.com/gifs/frog-muppets-funny-J0UgtZkOjXOe)
```
Comment (Abizern) [inline 1introduction/tutorial.md:68]: Oops - Broken link.

File: 1introduction/tutorial.md (hunk 1 above)
Comment (Abizern) [reply 1introduction/tutorial.md:17]: Actually - That's something the coach can help with if it appears.

**Comment by Abizern** [comment]
Possible Stretch Task

- Reset button to zero the counter.
- Autolayout the elements to handle rotation.
- Launch screen.

File: 1introduction/tutorial.md (hunk 1 above)
Comment (yvettecook) [reply 1introduction/tutorial.md:17]: @Abizern Will add in - incase people access it outside of the workshop and get confused. Do you have a screenshot of the popup I can use to illustrate?

File: 1introduction/tutorial.md (hunk 2 above)
Comment (yvettecook) [reply 1introduction/tutorial.md:68]: 🙈 Thanks! Will fix

File: 1introduction/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+  (Hint: To bring up the emoji keyboard, hit **ctrl** + **cmd** + **space**)
+
+  Run your app, and hit that button, and (fingers crossed) see your emoji appear.
+
+### 3 ... 2 ... 1 ... 🚀
+
+15. Finally, let's make the counter increment (good bye emojis, for now).
+
+  For this, our ViewController is going to need a local variable. Add this line to the top of your `ViewController`:
```
Comment (josephlord) [inline 1introduction/tutorial.md:183]: Would "property", "instance variable" or just "variable" be better than "local variable" as strictly it isn't a local variable? Sorry to be picky.

**Pull request merged**